SUI/SUI/Login/LoginView.swift

```diff
@@ -0,0 +1,216 @@
+// LoginView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран логина
+struct LoginView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let lightGrayColorName = "lightGray"
+        static let darkPurpleColorName = "darkPurple"
+        static let loginString = "Log in"
+        static let singUpString = "Sing up"
+        static let textFieldPlaceholderString = "+0(000)-000-00-00"
+        static let passwordTitle = "Password"
+        static let passwordPlaceholderString = "Введите пароль"
+        static let verificationButtonTitle = "CHECK VERIFICATION"
+        static let singUpButtonTitle = "SING UP"
+        static let alertTitle = "Помощь"
+        static let alertMessageString = "Пароль должен содержать от 6 до 15 символов"
+        static let alertButtonTitle = "Ok"
+        static let questionText = "Forgot your password?"
+        static let alertDoTitle = "Что делать?"
+        static let alertMessageDoString = "Позвонить по номеру +8 999 99 99 999"
+        static let logoViewHeight: CGFloat = 90
+        static let twoNumber: CGFloat = 2
+        static let phoneCountNumber = 18
+        static let titleCornerRadiusNumber: CGFloat = 45
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        ZStack {
+            VStack {
+                navigationBarView
+                Spacer(minLength: 50)
+                VStack {
+                    titleView
+                    phoneTextFieldView
+                    lineView
+                    passwordTitleView
+                    passwordTextFieldView
+                    lineView
+                }
+                VStack {
+                    verificationButtonView
+                    singUpButtonView
+                    singUpNavigationLinkView
+                    alertButtonView
+                    verificationNavigationLinkView
+                }
+            }
+        }
+        .navigationBarBackButtonHidden(true)
+    }
+
+    // MARK: - Private Properties
+
+    @StateObject private var loginViewModel = LoginViewModel()
+    @FocusState private var loginIsFocused: Bool
```

Comment:
```suggestion
     @StateObject private var loginViewModel = LoginViewModel()
    
      @FocusState private var loginIsFocused: Bool
     @FocusState private var passwordIsFocused: Bool
    
```
Отступ между разными враперами